sdk/communication/Azure.Communication.CallAutomation/src/Models/Misc/CallAutomationConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallAutomation.Models.Misc
```

Comment:
nit: we should try to avoid using namespace like 
Azure.Communication.CallAutomation.Models.*, but put everything in Azure.Communication.CallAutomation

Reply:
will update at next change.